tests/tests.js

```diff
@@ -4,42 +4,125 @@ const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
-
-    expect(price).to.equal(39.37)
+describe('pricing', () => {
+  describe('formatPrice', () => {
+    it('return the price with 2 decimal places', () => {})
+    let formatprice = pricing.formatPrice(33.323443)
+    expect(formatprice).to.equal(33.32)
```

Comment:
Syntax error, lines 10 and 11 aren't in this test which means the test isn't actually testing anything